plugins/kubernetes-backend/src/service/KubernetesBuilder.test.ts

```diff
@@ -779,6 +795,9 @@ metadata:
     expect(response.body).toMatchObject({
       permissions: [
         { type: 'basic', name: 'kubernetes.proxy', attributes: {} },
+        { type: 'basic', name: 'kubernetes.resource', attributes: {} },
+        { type: 'basic', name: 'kubernetes.service', attributes: {} },
```

Comment:
should these be plural?

Reply:
Sure, I can rename the permissions to plural if it makes more sense.

Reply:
But these are singular, because permission names are:
`kubernetes.service` permission and `kubernetes.resource` permission.

Reply:
@mclarke47 updated.